gwtmockito/src/main/java/com/google/gwtmockito/impl/StubGenerator.java

```diff
@@ -61,6 +62,14 @@
     STUB_METHODS.put(
         new ClassAndMethod(InputElement.class, "as"),
         new ReturnMockStubMethod(InputElement.class));
+	// URL.encodeQueryStringImpl
+	STUB_METHODS.put(
```

Comment:
Use 2 or 4 spaces for indents here instead of tabs, consistent with the rest of the file